tomviz/pvextensions/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+
+include(${PARAVIEW_USE_FILE})
+
+add_paraview_plugin(tomvizExtensions "1.0"
+  SERVER_MANAGER_XML TomVizExtensions.xml
```

Comment:
I was copying the DAX plugin's capitalization.  However, I later realized that the DAX plugin hasn't been used in ages and won't actually work with the current CMake setup...  But I didn't go back and fix stuff since it already worked.